src/read.c

```diff
@@ -1175,11 +1181,19 @@ void ReadFuncExpr (
 	    ASS_LIST( nams, narg+nloc, name );
 	    Match(S_IDENT,"identifier",S_RPAREN|S_LOCAL|STATBEGIN|S_END|follow);
 	}
+
+	if(TLS(Symbol) == S_DOTDOT) {
+	    SyntaxError("Three dots required for variadic argument list");
+	}
```

Comment:
Off-topic, but: Why do we sometimes start SyntaxError messages in upper case, and sometimes in lowercase?